news/feature-412.md

```diff
@@ -0,0 +1,15 @@
+grpc: Added service account option for grpc destinations.
```

Comment:
Let's name all the relevant destinations:

`bigquery()`, `google-pubsub-grpc()`